mri/operators/fourier/non_cartesian.py

```diff
@@ -348,7 +348,7 @@ def __init__(self, samples, shape, implementation='cpu', n_coils=1,
                              ' chosen. Please choose between "cpu" | "gpuNUFFT"')
 
     def op(self, data, *args):
-        """ This method calculates the masked non-cartesian Fourier transform
+        """ This method computes the masked non-uniform Fourier transform
```

Comment:
(D401) First line should be in imperative mood.
(D210) No whitespaces allowed surrounding docstring text
```suggestion
        """Compute the masked non-uniform Fourier transform
```